database/model/resumeModel.js

```diff
@@ -0,0 +1,135 @@
+const mongoose = require('mongoose');
+const db = require('../database.js');
+const { fieldsOfStudy } = require('./enums.js');
+
+const resumeSchema = new mongoose.Schema({
+  seekerId: { type: String, required: true },
+  firstName: {
+    type: String,
+    required: true,
+    trim: true,
+    minLength: 1,
+    maxLength: 40
+  },
+  lastName: {
+    type: String,
+    required: true,
+    trim: true,
+    minLength: 2,
+    maxLength: 40
+  },
+  email: {
+    type: String,
+    minLength: 5,
+    maxLength: 255,
+    required: true
+  },
+  phone: {
+    type: String,
+    validate: {
+      validator: function(v) {
+        return /\d{3}-\d{3}-\d{4}/.test(v);
+      },
+      message: props => `${props.value} is not a valid phone number!`
+    },
+    required: [true, 'User phone number required']
+  },
+  links: {
+    required: false,
+    type: {
+      linkedin: { type: String },
+      other: { type: String },
+    },
+  },
+  city: {
+    type: String,
+    required: false,
+    trim: true,
+    minLength: 2,
+    maxLength: 100
+  },
+  zipcode: {
+    type: String,
+    required: true,
+    minLength: 5,
+    maxLength: 10
```

Comment:
I was confused by this in some of the sample data I got. Why allow the range and not set it to a hard 5? Is it for the four digit extension plus hyphen? Allowing internationalization? If so, state, and county should be added as fields, too (state might should be added anyway).

Reply:
good point. I might just set it to 5.